.gitignore

```diff
@@ -25,3 +25,7 @@ src/.preload.php
 /phpunit.xml
 .phpunit.result.cache
 ###< phpunit/phpunit ###
+
+###> others ###
+.idea
```

Comment:
This rule should be defined in your personal configuration as it's related to an IDE and not the project itself. 😉

Reply:
done